### PR TITLE
[zh-cn] init translation of html attribute readonly

### PR DESCRIPTION
**Pull request by jasonren0403**
### Related issues and pull requests

Part of #17632

### CODE REVIEW

File: files/zh-cn/web/html/attributes/readonly/index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+当 **`readonly`** 布尔属性存在时，元素是不可变的，意味着用户无法编辑控件。
+
+{{EmbedInteractiveExample("pages/tabbed/attribute-readonly.html", "tabbed-shorter")}}
+
+## 概述
+
+如果在 input 元素上指定了 `readonly` 属性，由于用户无法编辑输入内容，因此该元素不参与约束验证。
+
+`{{HTMLElement("input/text", "text")}}`、`{{HTMLElement("input/search", "search")}}`、`{{HTMLElement("input/url", "url")}}`、`{{HTMLElement("input/tel", "tel")}}`、`{{HTMLElement("input/email", "email")}}`、`{{HTMLElement("input/password", "password")}}`、`{{HTMLElement("input/date", "date")}}`、`{{HTMLElement("input/month", "month")}}`、`{{HTMLElement("input/week", "week")}}`、`{{HTMLElement("input/time", "time")}}`、`{{HTMLElement("input/datetime-local"、 "datetime-local")}}`、"{{HTMLElement("input/number", "number")}}` 这些 `{{HTMLElement("input")}}` 类型和 `{{HTMLElement("textarea")}}` 表单控制元素均支持 `readonly` 属性。如果这些输入类型和元素中存在这个属性，则匹配 `{{cssxref(':read-only)}}` 伪类。如果不包含该属性，则将匹配 `{{cssxref(':read-write')}}` 伪类。
```
Comment (jasonren0403) [inline files/zh-cn/web/html/attributes/readonly/index.md:18]:
```suggestion
`{{HTMLElement("input/text", "text")}}`、`{{HTMLElement("input/search", "search")}}`、`{{HTMLElement("input/url", "url")}}`、`{{HTMLElement("input/tel", "tel")}}`、`{{HTMLElement("input/email", "email")}}`、`{{HTMLElement("input/password", "password")}}`、`{{HTMLElement("input/date", "date")}}`、`{{HTMLElement("input/month", "month")}}`、`{{HTMLElement("input/week", "week")}}`、`{{HTMLElement("input/time", "time")}}`、`{{HTMLElement("input/datetime-local"、 "datetime-local")}}`、`{{HTMLElement("input/number", "number")}}` 这些 `{{HTMLElement("input")}}` 类型和 `{{HTMLElement("textarea")}}` 表单控制元素均支持 `readonly` 属性。如果这些输入类型和元素中存在这个属性，则匹配 `{{cssxref(':read-only)}}` 伪类。如果不包含该属性，则将匹配 `{{cssxref(':read-write')}}` 伪类。
```

File: files/zh-cn/web/html/attributes/readonly/index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+当 **`readonly`** 布尔属性存在时，元素是不可变的，意味着用户无法编辑控件。
+
+{{EmbedInteractiveExample("pages/tabbed/attribute-readonly.html", "tabbed-shorter")}}
+
+## 概述
+
+如果在 input 元素上指定了 `readonly` 属性，由于用户无法编辑输入内容，因此该元素不参与约束验证。
+
+`{{HTMLElement("input/text", "text")}}`、`{{HTMLElement("input/search", "search")}}`、`{{HTMLElement("input/url", "url")}}`、`{{HTMLElement("input/tel", "tel")}}`、`{{HTMLElement("input/email", "email")}}`、`{{HTMLElement("input/password", "password")}}`、`{{HTMLElement("input/date", "date")}}`、`{{HTMLElement("input/month", "month")}}`、`{{HTMLElement("input/week", "week")}}`、`{{HTMLElement("input/time", "time")}}`、`{{HTMLElement("input/datetime-local"、 "datetime-local")}}`、`{{HTMLElement("input/number", "number")}}` 这些 `{{HTMLElement("input")}}` 类型和 `{{HTMLElement("textarea")}}` 表单控制元素均支持 `readonly` 属性。如果这些输入类型和元素中存在这个属性，则匹配 `{{cssxref(':read-only)}}` 伪类。如果不包含该属性，则将匹配 `{{cssxref(':read-write')}}` 伪类。
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/readonly/index.md:18]:
```suggestion
`{{HTMLElement("input/text", "text")}}`、`{{HTMLElement("input/search", "search")}}`、`{{HTMLElement("input/url", "url")}}`、`{{HTMLElement("input/tel", "tel")}}`、`{{HTMLElement("input/email", "email")}}`、`{{HTMLElement("input/password", "password")}}`、`{{HTMLElement("input/date", "date")}}`、`{{HTMLElement("input/month", "month")}}`、`{{HTMLElement("input/week", "week")}}`、`{{HTMLElement("input/time", "time")}}`、`{{HTMLElement("input/datetime-local", "datetime-local")}}`、`{{HTMLElement("input/number", "number")}}` 这些 `{{HTMLElement("input")}}` 类型和 `{{HTMLElement("textarea")}}` 表单控制元素均支持 `readonly` 属性。如果这些输入类型和元素中存在这个属性，则匹配 `{{cssxref(':read-only)}}` 伪类。如果不包含该属性，则将匹配 `{{cssxref(':read-write')}}` 伪类。
```

File: files/zh-cn/web/html/attributes/readonly/index.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+当 **`readonly`** 布尔属性存在时，元素是不可变的，意味着用户无法编辑控件。
+
+{{EmbedInteractiveExample("pages/tabbed/attribute-readonly.html", "tabbed-shorter")}}
+
+## 概述
+
+如果在 input 元素上指定了 `readonly` 属性，由于用户无法编辑输入内容，因此该元素不参与约束验证。
+
+`{{HTMLElement("input/text", "text")}}`、`{{HTMLElement("input/search", "search")}}`、`{{HTMLElement("input/url", "url")}}`、`{{HTMLElement("input/tel", "tel")}}`、`{{HTMLElement("input/email", "email")}}`、`{{HTMLElement("input/password", "password")}}`、`{{HTMLElement("input/date", "date")}}`、`{{HTMLElement("input/month", "month")}}`、`{{HTMLElement("input/week", "week")}}`、`{{HTMLElement("input/time", "time")}}`、`{{HTMLElement("input/datetime-local", "datetime-local")}}`、`{{HTMLElement("input/number", "number")}}` 这些 `{{HTMLElement("input")}}` 类型和 `{{HTMLElement("textarea")}}` 表单控制元素均支持 `readonly` 属性。如果这些输入类型和元素中存在这个属性，则匹配 `{{cssxref(':read-only)}}` 伪类。如果不包含该属性，则将匹配 `{{cssxref(':read-write')}}` 伪类。
```
Comment (yin1999) [inline files/zh-cn/web/html/attributes/readonly/index.md:18]:
```suggestion
`{{HTMLElement("input/text", "text")}}`、`{{HTMLElement("input/search", "search")}}`、`{{HTMLElement("input/url", "url")}}`、`{{HTMLElement("input/tel", "tel")}}`、`{{HTMLElement("input/email", "email")}}`、`{{HTMLElement("input/password", "password")}}`、`{{HTMLElement("input/date", "date")}}`、`{{HTMLElement("input/month", "month")}}`、`{{HTMLElement("input/week", "week")}}`、`{{HTMLElement("input/time", "time")}}`、`{{HTMLElement("input/datetime-local", "datetime-local")}}`、`{{HTMLElement("input/number", "number")}}` 这些 `{{HTMLElement("input")}}` 类型和 `{{HTMLElement("textarea")}}` 表单控制元素均支持 `readonly` 属性。如果这些输入类型和元素中存在这个属性，则匹配 `{{cssxref(':read-only')}}` 伪类。如果不包含该属性，则将匹配 `{{cssxref(':read-write')}}` 伪类。
```

**Review by yin1999 — COMMENTED** [review]
see: mdn/content#31359

**Review by yin1999 — APPROVED** [review]

**Pull request merged**